curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657ef609d771fa7dc0a15b17.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 657ef609d771fa7dc0a15b17
+title: Step 28
+challengeType: 20
+dashedName: step-28
+---
+
+# --description--
+
+In Python, the `if __name__ == "__main__":` block ensures that your script runs only when it is executed directly, not when imported as a module in another script. This is a common practice in Python to provide a starting point for a program.
+
+Here's an example of how you might use this:
+
+```py
+def some_function():
+    print("Function is running")
+
+if __name__ == "__main__":
+    some_function()
+
+```
+
+
+The `if __name__ == "__main__":` block needs to be written at the end of this expense tracker program. Inside this block, the main function is called. This will be the entry point of the expense tracker program. 
+
+This is the code that was added:
+
+```py
+if __name__ == "__main__":
+    main()
+```
+
+Now you can test out the full program in the Preview section on the right panel of this page.
```

Comment:
I would suggest removing or rephrasing this because the UI might change in the future or it may be readjusted to devices of different screen sizes and the panel may not be located to the right.

Reply:
I have removed this